Source/x11/XGServerEvent.m

```diff
@@ -455,6 +528,9 @@ - (void) processEvent: (XEvent *) event
             break;                /* Unknown button */
           }
 
+        if (menuButtonEnabled == NO && eventType == menuMouseButton)
```

Comment:
Shouldn't me add this further up where we decide which mouse event to send? we would need to duplicate the code and I am really not sure what is less confusing.

Reply:
What does confuse you here? I'd rather not insert return statement into if...else... block of code. Here return statement has good visibility even with code folding editor feature is used. Maybe I got you wrong?